enlace/programs/profile.php

```diff
@@ -818,7 +818,25 @@ function(response) {
     if ($sess[1] != $current_session) {
         $current_session = $sess[1];
         ?><tr><td class="all_projects" colspan="2">
-                                    <a href="javascript:;" onclick="$('.toggler_<?php echo $sess[0] ?>').toggle();"><?php echo $current_session; ?></a></td></tr>
+                                    <a href="javascript:;" onclick="$('.toggler_<?php echo $sess[0] ?>').toggle();"><?php echo $current_session; ?></a>
+                                    <?php if(mysqli_num_rows($program_info) == 0 &&
+                                             $sess[3] != '0000-00-00' && $sess[4] != '0000-00-00' &&
+                                             $sess[3] < $sess[4]) { ?>
```

Comment:
This could be a good place to check whether the program has days of the week selected.

Reply:
Unfortunately, that would require changing a bit more since the mysqli_fetch_row call above forces us to indices.  In general, I follow whatever code style is in the immediate surrounding area, so I'd want to change the whole thing or keep in line.  I think that I will be more aggressive with that once we have a better idea of the scope of the overall commitment Enlace has to the project.

Reply:
Ah, fair.  Makes sense!